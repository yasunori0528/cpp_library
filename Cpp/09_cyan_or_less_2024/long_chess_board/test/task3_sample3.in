2
2 901234567
1 141592653 1 626433832
2 890123456
1 197169399 2 502881693
