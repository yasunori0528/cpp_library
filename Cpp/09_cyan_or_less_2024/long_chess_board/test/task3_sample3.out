242420590
152856147
